JS2-week2/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Prototype</title>
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
Styles do not work, check the path to the file - is it correct?
Make sure everything works and run your code before making a PR :)